src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OtlpLogRecordTransformer.cs

```diff
@@ -49,7 +47,7 @@ public OtlpLogRecordTransformer(SdkLimitOptions sdkLimitOptions, ExperimentalOpt
             var otlpLogRecord = this.ToOtlpLog(logRecord);
             if (otlpLogRecord != null)
             {
-                var scopeName = logRecord.CategoryName ?? logRecord.Logger?.Name ?? DefaultScopeName;
+                var scopeName = logRecord.Logger.Name;
```

Comment:
In the tests, we are proving that the values of `logRecord.Logger.Name` and `logRecord.CategoryName` are the same. Is there any difference in using one over the other in this context?

Reply:
OTLPExporter can remain agnostic to ILogger specific `CategoryName`.

Reply:
No practical difference between the two. This is more of a philosophical change which brings OtlpExporter closer to the spec. The goal being it doesn't know anything about any specific logging framework (ILogger being one of those).